include/geojson/features/LineStringFeature.hpp

```diff
@@ -37,10 +37,10 @@ namespace geojson {
                 return boost::get<linestring_t>(this->geom);
             }
 
-            void visit(FeatureVisitor& visitor) {
+            void visit(FeatureVisitor& visitor) override {
                 visitor.visit(this);
             }
     };
 }
 
-#endif // GEOJSON_LINESTRING_FEATURE_H
```

Comment:
Not seeing any actual difference on these endif lines, I'm guessing maybe a newline?

Reply:
Yeah, it was a missing EOF newline.

![Screenshot 2024-06-03 at 7 40 05 AM](https://github.com/NOAA-OWP/ngen/assets/126464/ed36f91c-9af2-4ebf-bac9-06c662919059)

See that little 'do not enter' symbol in the red block? That's the indication Github uses.

Reply:
Interesting, the mobile app doesn't show the icon that the web view does...